src/pdf/DriverOverview/DriverOverview.tsx

```diff
@@ -82,10 +82,17 @@ const DriverOverview = async ({
     date,
 }: Props): Promise<React.ReactElement> => {
     const requiredData = await getRequiredData(parcelIds);
+    const { data, error } = await supabase
+        .from("website_data")
+        .select("name, value")
+        .eq("name", "driver_overview_message")
+        .single();
+    const message = error ? "" : data.value;
```

Comment:
If there's an error, should we say that in the message?

Reply:
yep makes sense